src/main/resources/tags/tall_flowers.json

```diff
@@ -0,0 +1,10 @@
+{
+	"values" : [
+		"minecraft:lilac",
+		"minecraft:rose_bush",
+		"minecraft:peony",
+		"minecraft:tall_grass",
+		"minecraft:large_fern",
+		"minecraft:sunflower"
+    ]
```

Comment:
I mean, a bit redundant since that tag already exists in Minecraft, at least in 1.16...
But it's fine I guess.

Reply:
how can you know when these tags were added?

Reply:
![image](https://user-images.githubusercontent.com/25465545/96373404-832f7e00-116c-11eb-8520-38d29fbc0698.png)

Reply:
oh i ddint even know that there were other tags there